docker-app/qfieldcloud/core/models.py

```diff
@@ -1063,6 +1062,28 @@ class PackagingOffliner(models.TextChoices):
         QGISCORE = "qgiscore", _("QGIS Core Offline Editing (deprecated)")
         PYTHONMINI = "pythonmini", _("Optimized Packager")
 
+    @property
+    def localized_layers(self) -> list[dict[str, Any]]:
+        """
+        Retrieve all layers from project_details that are marked as localized.
+
+        Returns:
+            A list of layer detail dictionaries where each dict has 'is_localized' == True.
+            If project_details is missing or empty, returns an empty list.
+        """
+        if not self.project_details:
+            return []
+
+        layers_by_id = self.project_details.get("layers_by_id", {})
+
+        localized = []
+        for layer_detail in layers_by_id.values():
+            is_local = layer_detail.get("is_localized", False)
+            if is_local:
```

Comment:
```suggestion
            if layer_detail.get("is_localized", False):
```

Do we really need the variable? If so, why don't we name it `is_localized` instead of renaming it to `is_local`?

Reply:
Changed!